tests/services/copy_project/test_copy_project.py

```diff
@@ -0,0 +1,54 @@
+import json
+
+from kili.client import Kili
+from kili.services.copy_project import CopyProject
+
+
+def test_copy_project_e2e_no_assets_no_labels():
```

Comment:
if this is an end to end test, this should go to the end 2 end tests. Here it is one, as it requires a running Kili server.